src/dbnode/client/session_fetch_bulk_blocks_test.go

```diff
@@ -1914,14 +1779,9 @@ type MockTChanNodes []*rpc.MockTChanNode
 func (c MockTChanNodes) expectFetchMetadataAndReturn(
 	result []testBlocksMetadata,
 	opts AdminOptions,
-	isV2 bool,
 ) {
 	for _, client := range c {
-		if isV2 {
-			expectFetchMetadataAndReturnV2(client, result, opts)
-		} else {
-			expectFetchMetadataAndReturn(client, result, opts)
-		}
+		expectFetchMetadataAndReturnV2(client, result, opts)
```

Comment:
nit: Maybe we can rename this from `expectFetchMetadataAndReturnV2` to just `expectFetchMetadataAndReturn`?